src/libPMacc/include/nvidia/functors/Assign.hpp

```diff
@@ -22,8 +22,6 @@
 
 #pragma once
 
-#include "pmacc_types.hpp"
```

Comment:
Sorry for that, I didn't know that.
At least the review process works in the end.